spec.md

```diff
@@ -1382,13 +1388,23 @@ For `200 OK` and `201 Created` response codes, the following fields are defined:
 Currently only shared devices are supported; a distributed file system which
 can be mounted on all app instances simultaneously.
 
-| Field | Type | Description |
+| Response Field | Type | Description |
 | --- | --- | --- |
 | volume_id* | string | ID of the shared volume to mount on every app instance. |
 | mount_config | object | Configuration object to be passed to the driver when the volume is mounted. |
 
 \* Fields with an asterisk are REQUIRED.
 
+##### Endpoint Object
+
+| Response Field | Type | Description |
+| --- | --- | --- |
+| host* | string | A host name or a single IP address. |
+| ports* | array of strings | A non-empty array. Each element is either a single port (for example "443") or a port range (for example "9000-9010"). |
+| protocol | string | The protocol. Valid values are `tcp`, `udp`, or `all`. The default value is `tcp`. |
```

Comment:
Would it make sense to allow L7 protocols like HTTP in this place? An implementation like the https://github.com/Peripli/istio-broker-proxy could decide on specific Istio rules based on that protocol information. The alternative would be to configure this out-of band and put it into Network Profile Data.

Reply:
I would rather keep this PR as simple as it is now and move additional topics to another PR.